src/modals/SettingsModal/WalletsSettingsSection.tsx

```diff
@@ -53,7 +54,7 @@ const WalletsSettingsSection = () => {
   const handleRemoveWallet = (walletName: string) => {
     WalletStorage.delete(walletName)
     AddressMetadataStorage.delete(walletName)
-    dispatch(walletDeleted(walletName))
+    dispatch(walletName === activeWalletName ? activeWalletDeleted() : walletDeleted(walletName))
```

Comment:
I dispatch a different action when the active wallet gets deleted than when a different wallet does so that I can clear the state in the first case, but not in the second.

Reply:
Well done.